_shows/16_17/di_and_viv_and_rose.md

```diff
@@ -59,8 +59,29 @@ assets:
     video: hHNbn9f
     title: Backstage At
 
+
+links:
+  - type: Review
+    href: http://www.hercampus.com/school/nottingham/review-di-and-viv-and-rose-nottingham-new-theatre
+    snapshot: x6O6D
+    publisher: HerCampus 
+    author: Emily Brady
+    title: "Review: Di and Viv and Rose at The Nottingham New Theatre"
+    date: 2017-03-28
+    quote: "The play is imbued with a fantastic energy – the dance numbers were particularly stand-out. The three actresses drunkenly dancing after a night out was a highlight of the show. But it didn’t shy away from quiet moments – the ending was particularly poignant, and I left with a definite lump in my throat. "
+  - type: Review
+    href: http://www.impactnottingham.com/2017/03/di-viv-rose-nnt/
+    snapshot: BeN8I
+    publisher: Impact Nottingham
+    author: Daniel McVey
+    title: "Di, Viv and Rose @ Nottingham New Theatre"
+    date: 2017-03-25
+    rating: 8/10
+    quote: "The play is remarkable in that, despite the abrupt and extreme situations that it encompasses, it still overall feels realistic, especially in its portrayal of the friendship between Di, Viv and Rose. In fact, I would even suggest that, despite the hardships that each of the characters face, as an audience we can’t help but see similarities between the characters and our own circle of close friends. "
```

Comment:
Is quite long. Would suggest trimming to:
"The play is remarkable in that, despite the abrupt and extreme situations that it encompasses, it still overall feels realistic, especially in its portrayal of the friendship between Di, Viv and Rose. [A]s an audience we can’t help but see similarities between the characters and our own circle of close friends."
Also trim whitespace, regardless.